test/bugfixes/test_bugfixes.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from tinygrad.tensor import Tensor
+from tinygrad.helpers import dtypes
+import numpy as np
+
+class TestBugfixes(unittest.TestCase):
+  def test_1479(self):
+    # Just should compile
+    cond = Tensor([0,1]) == Tensor([1,1])
+    np.testing.assert_allclose(cond.where(Tensor([1,1], dtype=dtypes.bool), 0).numpy(), [0, 1])
```

Comment:
Should probably just throw this with the rest of the where tests in `test_ops.py` just add a comment to #1479.

Reply:
done